modules/client/test/integration/client/methods.test.ts

```diff
@@ -1628,7 +1624,7 @@ describe("Client", () => {
         expect(plugins[0].current.release.metadata).toMatchObject(
           releaseMetadata,
         );
-        expect(plugins[0].releases.length).toBe(2)
+        expect(plugins[0].releases.length).toBe(2);
```

Comment:
Is the SDK repo not linting / format checking itself? This should be a task for the future - not for now.

Reply:
Yep, createed a PR for that https://aragonassociation.atlassian.net/browse/OS-778